core/scheduler/scheduler.go

```diff
@@ -245,9 +245,17 @@ func (s *Scheduler) resolveDuties(ctx context.Context, slot slot) error {
 		return err
 	}
 
-	err = s.resolveProDuties(ctx, slot, vals)
-	if err != nil {
-		return err
+	// TODO: replace if false with check for builder api flag
```

Comment:
This would be some check on the builder api flag not sure how we want to handle it for now

Reply:
yeah, I think you can just add a `builderAPI bool` variable/field to `New` and `Scheduler`. We can add the flag itself in another PR. But it would basically be adding `BuilderAPI bool` field to `app.Config` and wiring it up in the `cmd` and `app` packages.